component/component.css

```diff
@@ -94,4 +94,28 @@
     /* Image responsiveness */
     --max-inline-size-img: 100%;
     /* *************************************** */
+}
+
+@media only screen and (max-width: 930px) {
+    html {
+        font-size: 50%;
+    }
+}
+
+@media only screen and (max-width: 768px) {
+    html {
+        font-size: 45%;
+    }
+}
+
+@media only screen and (max-width: 640px) {
+    html {
+        font-size: 35%;
+    }
+}
+
+@media only screen and (max-width: 480px) {
+    html {
+        font-size: 25%;
+    }
 }
```

Comment:
you may have these in utility or base CSS although it depends on your file structure. Just make sure you use the same media queries for all to improve reusability. Also, check the LMS Figma file for more break pts, if you feel the need.